Tools/Scripts/libraries/webkitcorepy/webkitcorepy/__init__.py

```diff
@@ -77,12 +77,12 @@
     AutoInstall.register(Package('packaging', Version(20, 4)))
 
 AutoInstall.register(Package('pyparsing', Version(2, 4, 7)))
-AutoInstall.register(Package('requests', Version(2, 24)))
+AutoInstall.register(Package('requests', Version(2, 31, 0)))
```

Comment:
Looks like this version doesn't support Python 2. That's fine in general, we just should let Python 2 use the old version still.

Reply:
Oops. At least that's easy to fix.